src/components/form/form.style.js

```diff
@@ -43,26 +43,22 @@ export const StyledLeftButtons = styled.div`
   }
 `;
 
-const FormButtonAnimation = keyframes`
-  0%   { bottom: -50px; }
-  100% { bottom: 0; }
-`;
-
 export const StyledFormFooter = styled.div`
```

Comment:
I suggest to list all props at the beginning of the Styled component like
```js
export const StyledFormFooter = styled.div`
${({buttonAlignment, stickyFooter, theme}) =>css`
//rest of the code
` }
```

Reply:
Thanks Dawid, maybe that does need to be done at some point but it's not really related to my changes here so I'm just going to leave it as it is for now 👍🏻